src/sdk/cpp/lib/src/iotea.cpp

```diff
@@ -293,22 +285,25 @@ CallToken EventContext::Call(const Callee& callee, const json& args, const int64
     return CallToken{call_id, timeout};
 }
 
+int64_t EventContext::GetNowMs() const {
+    auto now = std::chrono::steady_clock::now().time_since_epoch();
+    return std::chrono::duration_cast<std::chrono::milliseconds>(now).count();
+}
+
 //
 // CallContext
 //
 CallContext::CallContext(const std::string& talent_id, const std::string& channel_id, const std::string& feature,
-                         const Event& event, call_handler_ptr call_handler, publisher_ptr publisher, uuid_generator_func_ptr uuid_gen)
-    : EventContext{talent_id, channel_id, event.GetSubject(), event.GetReturnTopic(), call_handler, publisher, uuid_gen}
+                         const Event& event, reply_handler_ptr reply_nandler, publisher_ptr publisher, uuid_generator_func_ptr uuid_gen)
```

Comment:
nandler?

Reply:
You sure have an eye for detail :) Will fix.